lib/perl/Genome/InstrumentData/Composite/Workflow/Generator/AlignAndMerge.pm

```diff
@@ -72,6 +82,40 @@ sub generate {
     return $workflows, $inputs;
 }
 
+sub _get_lsf_resource_string_for_aligner_and_instrument_data {
+    my $class = shift;
+    my $aligner_name = shift;
+    my @instrument_data = @_;
+
+    my $merged_result_class = Genome::InstrumentData::Command::AlignAndMerge->merged_result_class($aligner_name);
+    my $estimated_gtmp_bytes = sum(map { $merged_result_class->estimated_gtmp_for_instrument_data($_) } @instrument_data);
```

Comment:
I think we should add some error checking here in case `estimated_gtmp_for_instrument_data` isn't defined on the `$merged_result_class`. It might be sufficient to just define an abstract version of `estimated_gtmp_for_instrument_data` on `G::ID::AR::Merged` and have that error out when it's not defined on the subclass.